src/app/large-display/large-display.component.html

```diff
@@ -1,32 +1,22 @@
-<ng-container [ngSwitch]="currentScreen">
+<ng-container [ngSwitch]="currentScreen$ | async">
   <ng-container *ngSwitchCase="ScreenId.Lobby">
-    <div class="large-display-content lobby">
-      <p>Waiting for the round to start&hellip;</p>
-    </div>
-    
-    <app-bottom-bar>
-      <div right>
-        <button mat-raised-button color="accent" (click)="startRound()">
-          Start Round
-        </button>
-      </div>
-    </app-bottom-bar>
+    <app-session-lobby></app-session-lobby>
```

Comment:
Since we're doing this, we should probably remove the routing to the Session Lobby Component (Especially because the Start Round button doesn't seem to work when the Session Lobby Component is not part of the Large Display Component)

Reply:
Roger that!